src/bootstrap/config.rs

```diff
@@ -582,6 +586,7 @@ impl Config {
         };
         config.rust_debuginfo_lines = debuginfo_lines.unwrap_or(default);
         config.rust_debuginfo_only_std = debuginfo_only_std.unwrap_or(default);
+        config.rust_debuginfo_tools = debuginfo_tools.unwrap_or(default);
```

Comment:
This `default` is true on stable/beta/nightly; could you either move it up to where we have `llvm_assertions` or change the default to false?

Reply:
OK, I'll make it plain false.